packages/react-server/core/config.js

```diff
@@ -10,12 +10,18 @@ if (SERVER_SIDE) {
 	module.exports = function () {
 		// only read out the config once, and then cache it. -sra.
 		if (null === config) {
-			/*eslint-disable no-process-env */
-			if (process.env.REACT_SERVER_CONFIGS) {
+
+			//eslint-disable-next-line no-process-env
+			if (process.env.REACT_SERVER_CONFIG_SOURCE) {
+				var path = require('path');
+				//eslint-disable-next-line no-process-env
+				config = require(path.join(process.cwd(), process.env.REACT_SERVER_CONFIG_SOURCE));
+			}
+			//eslint-disable-next-line no-process-env
+			else if (process.env.REACT_SERVER_CONFIGS) {
 				var fs = require("fs");
-				/*eslint-disable no-process-env */
+				//eslint-disable-next-line no-process-env
 				var configFile = fs.readFileSync(process.env.REACT_SERVER_CONFIGS + "/config.json");
-				/*eslint-disable no-process-env */
```

Comment:
`eslint-disable` in original code actually disable eslint check for all below code. So, I just update it with `eslint-disable-next-line`.